utilities/Constants.ts

```diff
@@ -30,7 +30,7 @@ export const COJ_DECLARATIONS = [
   {
     id: "isDeclareProvisional",
     label:
-      "I understand that programme and post allocations are provisional and subject to change until confirmed by HEE WM and/or my employing organisation."
+      "I understand that programme and post allocations are provisional and subject to change until confirmed by HEE/NES/ HEIW/NIMDTA and/or my employing organisation."
```

Comment:
The GG does have a space between `NES` and `NEIW` but that seems like an error compared to other usages, so we can remove that extra whitespace here.